RELEASE.md

```diff
@@ -38,21 +40,24 @@ For production releases, use the dedicated PyPI workflow:
 
 ### Step-by-Step Process
 
-1. **Update Version** (Manual):
+1. **Update Version and Changelog** (Manual):
    ```bash
+   # Update Changelog in README.md
    # Update version in pyproject.toml and package.json
    # Example: Change from "5.2.1.dev0" to "5.2.1"
    ```
 
 2. **Commit and Tag**:
    ```bash
-   git add pyproject.toml package.json
+   # Commit changes
+   git add pyproject.toml package.json README.md
    git commit -m "Bump version to 5.2.1"
-   git push origin main
    
-   # Create and push tag
+   # Create tag
    git tag v5.2.1
-   git push origin v5.2.1
+
+   # Push changes and tag
+   git push upstream main --tags
```

Comment:
Hi Rosio, since you are a maintainer on the repo and can freely make new branches, is it needed to keep your fork and the origin/upstream complexity?